docassemble/EFSPIntegration/efm_client.py

```diff
@@ -70,15 +70,20 @@ def user_visible_resp(resp):
       except:
         return ApiResponse(resp.status_code, resp.text, None)
 
-    def AuthenticateUser(self, email:str=None, password:str=None, jeffnet_token:str=None):
+    def AuthenticateUser(self, 
+        tyler_email:str=get_config('efile proxy').get('tyler email'), 
+        tyler_password:str=get_config('efile proxy').get('tyler password'), 
+        jeffnet_key:str=get_config('efile proxy').get('jeffnet api token')):
```

Comment:
Same question as above. Not sure it's an issue--but if it is, the workaround is to make the default value None, test for `is None`, and assign a default value in the body of the method or function.

Reply:
Good call, I forgot about that issue. Easy change though.